models/evc.py

```diff
@@ -1201,51 +1200,27 @@ def check_list_traces(list_circuits):
         if not list_circuits:
             return {}
         uni_list = []
-        circuit_data = {}
-        for circuit_id in list_circuits:
-            circuit = list_circuits[circuit_id]
-            # if a inter-switch EVC does not have current_path, it does not
-            # make sense to run sdntrace on it
-            if not circuit.is_intra_switch() and not circuit.current_path:
-                continue
+        for circuit in list_circuits:
             uni_list.append(circuit.uni_a)
             uni_list.append(circuit.uni_z)
-            interface_a = uni_to_str(circuit.uni_a)
```

Comment:
@gretelliz since this `utils.uni_to_str` is no longer used we can also nuke the definion from `utils.py` can you double check this?

Reply:
@viniarck, you are right.